test/performance/odp_l2fwd.c

```diff
@@ -91,6 +91,7 @@ typedef struct {
 	int sched_mode;         /* Scheduler mode */
 	int num_groups;         /* Number of scheduling groups */
 	int burst_rx;           /* Receive burst size */
+	int pools;              /* Pools per interface option */
```

Comment:
Naming of this variable makes reading the code a bit confusing. E.g. `if (gbl_args->appl.pools == 0)` -> use multiple pools. For example pool_per_if would be easier to understand.

Reply:
fixed in v2